SU2_CFD/src/CMarkerProfileReaderFVM.cpp

```diff
@@ -428,13 +459,68 @@ void CMarkerProfileReaderFVM::WriteMarkerProfileTemplate() {
       /*--- Access the default data for this marker. ---*/
 
       string Marker_Tag = profileTags[iMarker];
-
+      su2double p_total   = config->GetInlet_Ptotal(Marker_Tag);
+      su2double t_total   = config->GetInlet_Ttotal(Marker_Tag);
+      auto flow_dir = config->GetInlet_FlowDir(Marker_Tag);
+      
       /*--- Header information for this marker. ---*/
 
       node_file << "MARKER_TAG= " << Marker_Tag              << endl;
       node_file << "NROW="        << numberOfRowsInProfile[iMarker] << endl;
       node_file << "NCOL="        << nColumns          << endl;
 
+      /*--- start of the header line (names of the columns) --- */
+
+      if (dimension==2)  
+        node_file << "# COORD-X  " << setw(24) << "COORD-Y    " << setw(24);
+      else 
+        node_file << "# COORD-X  " << setw(24) << "COORD-Y    " << setw(24) << "COORD-Z    " << setw(24); 
+
+      INLET_TYPE Kind_Inlet = config->GetKind_Inc_Inlet(Marker_Tag);
+
+      if (compressible) {
+        node_file << "TEMPERATURE" << setw(24) << "PRESSURE   " << setw(24); 
+      }
+      else {
+        switch (Kind_Inlet) {
+
+          case INLET_TYPE::VELOCITY_INLET:
+            node_file << "TEMPERATURE" << setw(24) << "VELOCITY   " << setw(24); 
+            break;
+          case INLET_TYPE::PRESSURE_INLET:
+            node_file << "TEMPERATURE" << setw(24) << "PRESSURE   " << setw(24); 
+            break;
+          default:
+            SU2_MPI::Error("Unsupported INC_INLET_TYPE.", CURRENT_FUNCTION);
+            break;
+        }
+      }
+
+      if (dimension==2)  
+        node_file << "NORMAL-X   " << setw(24) << "NORMAL-Y   " << setw(24);
+      else 
+        node_file << "NORMAL-X   " << setw(24) << "NORMAL-Y   " << setw(24) << "NORMAL-Z   " << setw(24);
+
+      switch (numberOfTurbVars) {
+        case 0:
+          /*--- no turbulence model---*/
+          break;
+        case 1:
+          /*--- 1-equation turbulence model: SA ---*/
+          turb_val[0] = config->GetNuFactor_FreeStream()*config->GetViscosity_FreeStreamND()/config->GetDensity_FreeStreamND();
+          node_file << "NU_TILDE   " << setw(24);
+          break;
+        case 2:
```

Comment:
This type of assumption makes it hard to extend the code, always prefer text (i.e. enums) to numbers.

Reply:
so maybe introduce ONE_EQUATION_MODEL, TWO_EQUATION_MODEL? Because I do not like having SA_model1 OR SA_model2 OR SA_model256 everywhere when the only thing we want to know is the number of turbulence variables.

Reply:
I made an update, but now we have case SA:case SA_NEG etc... here.

Reply:
I do not like either -> #770 
It is for that reason we have methods in CConfig (and some in option_structure) like GetSolid_Wall, GetViscous_Wall, GetFluidProblem, GetStructuralProblem, etc.
Like I've said in that issue that no one cared about, repeating that kind of switch statement is one of the main causes of bugs we have in SU2.